### PR TITLE
Barebones of Chiazo's Portfolio

### PR DESCRIPTION
**Pull request by chiazo**
I've set up all the elements needed and plan to go back to implement some javascript features and overall CSS.

### CODE REVIEW

File: portfolio/src/main/webapp/style.css [hunk 1]
```diff
@@ -1,9 +1,79 @@
+@import url('https://fonts.googleapis.com/css2?family=Cabin&family=Raleway:wght@500&display=swap');
+body {
+    /* background-color: rgb(102, 194, 143); */
+    /* font-family: 'Raleway', sans-serif; */
```
Comment (travis-miller) [inline portfolio/src/main/webapp/style.css:4]: It is usually better to delete commented code but if you think you will use it in the near future feel free to keep it in.

File: portfolio/src/main/webapp/index.html [hunk 2]
```diff
@@ -1,18 +1,105 @@
...
+			</div>
+			<div class="links">
+				<ul>
+					<li><i><a href="https://www.linkedin.com/in/chiazo"><img src="/images/favicons/linkedin.png"/></a></i>
+					</li>
+					<li><i><a href="https://github.com/chiazo"><img src="/images/favicons/github.png"/></a></i></li>
+					<li><i><a href="https://chiazo.github.io"><img src="/images/favicons/website.png"/></a></i></li>
+				</ul>
+			</div>
+		</div>
```
Comment (travis-miller) [inline portfolio/src/main/webapp/index.html:91]: This bit here could use a run of the code formatter.

File: portfolio/src/main/webapp/script.js [hunk 3]
```diff
@@ -15,14 +15,42 @@
...
+  const fact = facts[Math.floor(Math.random() * facts.length)];
 
   // Add it to the page.
-  const greetingContainer = document.getElementById('greeting-container');
-  greetingContainer.innerText = greeting;
+  const factContainer = document.getElementById('fact-container');
+  factContainer.innerText = fact;
 }
+
+function filterProjects(tag) {
```
Comment (jayceb04) [inline portfolio/src/main/webapp/script.js:30]: Would you mind adding a comment above the function definition about what this function does.

File: portfolio/src/main/webapp/script.js [hunk 4]
```diff
@@ -15,14 +15,42 @@
 /**
  * Adds a random greeting to the page.
  */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+function addRandomFact() {
```
Comment (jayceb04) [inline portfolio/src/main/webapp/script.js:18]: I see you've implemented this, but it's not used in index.html. Just wanted to check if that was intentional?

Comment (chiazo) [reply portfolio/src/main/webapp/script.js:18]: I actually edited this but I'm planning to delete it.

File: portfolio/src/main/webapp/script.js (hunk 3 above)
Comment (chiazo) [reply portfolio/src/main/webapp/script.js:30]: done!

File: portfolio/src/main/webapp/index.html (hunk 2 above)
Comment (chiazo) [reply portfolio/src/main/webapp/index.html:91]: Ok, I've formatted it again!

File: portfolio/src/main/webapp/style.css (hunk 1 above)
Comment (chiazo) [reply portfolio/src/main/webapp/style.css:4]: Deleted!

**Pull request merged**